getting-started.md

```diff
@@ -1,10 +1,10 @@
 ## Besu (Execution Client)
 
-Welcome to the Merged Besu Ethereum Mainnet Consensus Layer Client
+Welcome to your Post Besu Execution Layer Client
```

Comment:
```suggestion
Welcome to the Merged Besu Ethereum Mainnet Execution Layer Client
```
Not fixed and we need to say mainnet ethereum since we have on all packages so far, or at least I have when i was assigned a couple months ago.